check_pgactivity

```diff
@@ -7203,15 +7206,18 @@ SELECT
     seq.seqmax AS max_value,
     seq.seqincrement AS increment_by
    FROM pg_sequence seq
-   INNER JOIN pg_catalog.pg_depend d ON d.objid=seq.seqrelid
+   LEFT JOIN (SELECT d.objid, nspname, relname, attname, typname FROM
```

Comment:
nit: the FROM should be on the next line.

Reply:
Fixed in a forthcoming commit

Reply:
done in 73caa656410f68e02c2dcbc4499b3bd3add77fa2